tests/utils/test_utils.py

```diff
@@ -78,3 +78,10 @@ def test_find_first_and_second_and_third_level(self):
         assert self.mlp.activations[2] in found
         assert self.mlp.activations[1].children[0] in found
         assert self.mlp.activations[3].children[0].children[0]
+
+
+def test_imports():
```

Comment:
I do, but it is tricky because of the way how nosetests works. There is no guarantee which order the tests run and it uses the same python interpreter. Therefore theano might be still loaded. Not sure how to solve this.